aries_cloudagent/holder/routes.py

```diff
@@ -274,6 +274,10 @@ async def credentials_remove(request: web.BaseRequest):
         async with context.profile.session() as session:
             holder = session.inject(IndyHolder)
             await holder.delete_credential(credential_id)
+            topic = "acapy::record::credential::delete"
+            await session.profile.notify(
+                topic, {"id": credential_id, "state": "deleted"}
+            )
```

Comment:
```suggestion
        topic = "acapy::record::credential::delete"
        await context.profile.notify(
            topic, {"id": credential_id, "state": "deleted"}
        )
```

Better to close the session first.

Reply:
Great catch! I still have trouble understanding what's in the various base objects (like context and session), so this helped a lot